app/models/family_content_page.rb

```diff
@@ -7,7 +7,14 @@ def self.fae_fields
     {
       introduction: { type: Fae::TextArea },
       kicker: {type: Fae::TextArea},
-      body: { type: Fae::TextArea }
+      body: { type: Fae::TextArea },
+      start_topic: {type: Fae::TextField},
+      completed_topic: {type: Fae::TextField},
+      success_text: {type: Fae::TextField},
+      save_button: {type: Fae::TextField},
+      final_result_title: {type: Fae::TextField},
+      results: {type: Fae::TextField},
+      results_link: {type: Fae::TextField}
```

Comment:
I'm not sure if I'm using the best name for these new variables

Reply:
These look fine, maybe just a couple of suggestions:

* For the `results_link` it might be worth renaming it to `results_link_text` as the actual link is hardcoded (`/results/index`) but this represents the text we show for that link
* For `start_topic` and `completed_topic` maybe change to `start_topic_link_text` and `completed_topic_link_text`